docs/01_manual.md

```diff
@@ -81,81 +81,101 @@ For details about the service API see the [OpenAPI specification].
+      -f, --format  <arg>   stream item packaged in this format (tar|zip)
+          --help            Show help message
+    
+     trailing arguments:
+      item-id (required)   item-id of the item to stream
+    ---
+    
+    Subcommand: enum - Enumerates bags or Files
+      -a, --all                   enumerate all bags, including inactive ones
+      -e, --exclude-directories   enumerate only regular files, not directories
+      -d, --inactive              only enumerate inactive bags
+          --help                  Show help message
     
      trailing arguments:
-      item-id (required)        ID of the Bag or File to retrieve
-      <output-dir> (required)   directory in which to put the Bag or File
+      <bagId> (not required)   bag of which to enumerate the Files
     ---
     
-    Subcommand: enum - Enumerates Bags or Files
-      -a, --all        enumerate all Bags, including inactive ones
-      -d, --inactive   only enumerate inactive Bags
-          --help       Show help message
+    Subcommand: locate - Locates the item with <item-id> on the file system
+      -f, --file-data-location   resolve to file-data-location
+          --help                 Show help message
     
      trailing arguments:
-      <bagId> (not required)   Bag of which to enumerate the Files
+      <item-id> (required)   the item to locate
     ---
     
-    Subcommand: deactivate - Marks a Bag as inactive
+    Subcommand: deactivate - Marks a bag as inactive
           --help   Show help message
     
      trailing arguments:
-      <bag-id> (required)   Bag to mark as inactive
+      <bag-id> (required)   bag to mark as inactive
     ---
     
-    Subcommand: reactivate - Reactivates an inactive Bag
+    Subcommand: reactivate - Reactivates an inactive bag
           --help   Show help message
     
      trailing arguments:
-      <bag-id> (required)   Inactive Bag to re-activate
+      <bag-id> (required)   inactive bag to re-activate
     ---
     
-    Subcommand: prune - Removes Files from Bag, that are already found in 
-    reference Bags, replacing them with fetch.txt references
+    Subcommand: prune - Removes Files from bag, that are already found in reference bags, replacing them with fetch.txt references
           --help   Show help message
     
      trailing arguments:
-      <bag-dir> (required)         Bag directory to prune
-      <ref-bag-id>... (required)   One or more bag-ids of Bags in the BagStore to
+      <bag-dir> (required)         bag directory to prune
+      <ref-bag-id>... (required)   One or more bag-ids of bags in the bag store to
                                    check for redundant Files
     ---
     
-    Subcommand: complete - Resolves fetch.txt references from the BagStore 
-    and copies them into <bag-dir>
-          --help   Show help message
+    Subcommand: complete - Resolves fetch.txt references from the bag store and copies them into <bag-dir>
```

Comment:
Added option to keep the `fetch.txt` file, if present. This will leave the retrieved bag in a formally incomplete state (to the letter of the BagIt spec&mdash;the bagit tool doesn't recognize it).